master/internal/trial.go

```diff
@@ -348,9 +363,12 @@ func (t *trial) runningReceive(ctx *actor.Context) error {
 
 	case killTrial:
 		if t.task != nil {
-			if msg.timedOut {
-				ctx.Log().Info("killing unresponsive trial after timeout")
-			}
+			ctx.Tell(t.rp, scheduler.TerminateTask{TaskID: t.task.ID, Forcible: true})
+		}
+
+	case terminateTimeout:
+		if t.task != nil && msg.resets == t.resets {
+			ctx.Log().Info("killing unresponsive trial after timeout")
```

Comment:
non-blocking: we should add a trial ID :)

Reply:
sure

Reply:
The ID is already included as a field on all messages by this actor (once it gets an ID).

Reply:
Good call, here's an example:
```
INFO[2020-06-23T17:25:49-06:00] killing unresponsive trial after timeout      experiment-id=34 id=cfca40ef-6d0d-4dd0-8c8c-7f86826699cd system=master trial-id=34 type=trial
```